projects/kit/src/lib/masks/number/number-mask.ts

```diff
@@ -136,11 +137,16 @@ export function maskitoNumberOptionsGenerator({
             }),
         ],
         postprocessors: [
-            createMinMaxPostprocessor({decimalSeparator, min, max, minusSign}),
+            createMinMaxPostprocessor({
+                min,
+                max,
+                minusSign,
+                decimalSeparator: precision ? decimalSeparator : '',
+            }),
             maskitoPrefixPostprocessorGenerator(prefix),
             maskitoPostfixPostprocessorGenerator(postfix),
             createThousandSeparatorPostprocessor({
-                decimalSeparator,
+                decimalSeparator: precision ? decimalSeparator : '',
```

Comment:
This repeats 3 times. Destructured params can be overwritten, I believe (as if they were `let { xxx } = `), let's do it

Reply:
Fixed in https://github.com/taiga-family/maskito/pull/1908/commits/33e335a5f49b3dcbc46fcff9d4c13e6a3390e663